marc_to_solr/lib/traject_config.rb

```diff
@@ -1352,11 +1375,27 @@
   # these fields are no longer necessary
   context.output_hash.delete('name_title_100')
   context.output_hash.delete('name_title_100_vern')
-  context.output_hash.delete('uniform_240')
-  context.output_hash.delete('uniform_240_vern')
   context.output_hash.delete('name_title_245a')
   context.output_hash.delete('name_title_245a_vern')
   context.output_hash.delete('name_title_ae_s')
+  context.output_hash.delete('uniform_240')
+  context.output_hash.delete('uniform_240_vern')
+end
+
+# Creates uniform_title_1display for Uniform title display for works that do not have an author
+each_record do |_record, context|
+  doc = context.output_hash
+  uniform_t = []
+  search_field = []
+
+  uniform_t << doc['uniform_130'] if doc['uniform_130']
+  uniform_t << doc['uniform_130_vern'] if doc['uniform_130_vern']
+
+  context.output_hash['uniform_title_1display'] = [uniform_t.to_json.to_s] unless uniform_t.empty?
+
+  # these fields are no longer necessary
```

Comment:
do you need to remove this comment?

Reply:
I didn't think so - it's explaining why the fields are being removed (same as 1375)